packages/ember-routing/lib/services/router.js

```diff
@@ -0,0 +1,48 @@
+/**
+@module ember
+@submodule ember-routing
+*/
+
+import {
+  Service,
+  readOnly
+} from 'ember-runtime';
+import { get, descriptor } from 'ember-metal';
+import RouterDSL from '../system/dsl';
+
+/**
+   The Router service is the public API that provides component/view layer
+   access to the router.
+
+   @public
+   @class RouterService
```

Comment:
Should add the `@category ember-routing-router-service` flag here too (so that while the feature is pending it isn't listed in emberjs.com/api).